presto/presto.go

```diff
@@ -1359,11 +1426,11 @@ type NullTime struct {
 
 // Scan implements the sql.Scanner interface.
 func (s *NullTime) Scan(value interface{}) error {
-	switch value.(type) {
+	switch t := value.(type) {
```

Comment:
use `valueType` instead of `t`. (Yeah I know golang likies single letter variables. I don't :p)

Reply:
Sorry but I need to push back here. The var we instantiate here is not the type but of specific type so `typedValue` would be slightly better. But in both cases it's actually a time so a more specific name is even better. But `time` is the name of the package, hence `t`. This is idiomatic, `t` is used a lot in the examples from stdlib: https://golang.org/pkg/time/

Reply:
fair :) Let's leave t.